components/RecordFile.js

```diff
@@ -1,27 +1,27 @@
 import styled from 'styled-components';
 
-export default function RecordFile({ record, onHandleChange }) {
+export default function RecordFile({ record, onChange }) {
   return (
     <Record>
-      <Cover src={record.Cover} />
+      <Cover src={record.basic_information?.thumb} />
       <div>
         <BookmarkIcon
           type="checkbox"
           checked={record.isChecked}
-          onChange={() => onHandleChange(record.CatalogId)}
+          onChange={() => onChange(record.id)}
```

Comment:
you might want to rename the toggling function to something more specific and thus prevent having an onChange event that calls a function called `onChange`.

```suggestion
          onChange={() => onToggleBookmark(record.id)}
```